### PR TITLE
JBPM-8447: Unable to display task with task input equal to constant

### PR DESCRIPTION
**Pull request by pefernan**
@domhanak @cristianonicolai @nmirasch could you please take a look?

### CODE REVIEW

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Comment by cristianonicolai** [comment]
@domhanak @jstastny-cz  could you please take a look, thanks

File: jbpm-wb-forms/jbpm-wb-forms-backend/src/test/java/org/jbpm/workbench/forms/display/backend/conversion/TaskDataConverterUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        testSupportedConstantValues(false);
+    }
+
+    @Test
+    public void testSupportedConstantValuesWithTypeAlias() {
+        testSupportedConstantValues(true);
+    }
+
+    @Test
+    public void testSupportedconstantValuesWithParsingErrors() {
```
Comment (jstastny-cz) [inline jbpm-wb-forms/jbpm-wb-forms-backend/src/test/java/org/jbpm/workbench/forms/display/backend/conversion/TaskDataConverterUtilTest.java]:
```suggestion
    public void testSupportedConstantValuesWithParsingErrors() {
```

**Comment by pefernan** [comment]
@jstastny-cz solved thanks!

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
@jstastny-cz happy to approve this PR?

**Comment by pefernan** [comment]
@domhanak your turn!

**Comment by pefernan** [comment]
@tomasdavidorg thank you! proceding to merge

**Pull request merged**